client/src/containers/KsqlDB/KsqlDBList/KsqlDBList.jsx

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Header from '../../Header/Header';
+import { getSelectedTab } from '../../../utils/functions';
+import { Link } from 'react-router-dom';
+import KsqlDBInfo from './KsqlDBInfo/KsqlDBInfo';
+import KsqlDBStreams from './KsqlDBStreams/KsqlDBStreams';
+import KsqlDBTables from './KsqlDBTables/KsqlDBTables';
+import KsqlDBQueries from './KsqlDBQueries/KsqlDBQueries';
+
+class KsqlDBList extends Component {
+  state = {
+    clusterId: this.props.history.clusterId || this.props.match.params.clusterId,
+    ksqlDBId: this.props.history.ksqlDBId || this.props.match.params.ksqlDBId,
+    selectedTab: 'streams',
+    roles: JSON.parse(sessionStorage.getItem('roles')),
+  };
+
+  tabs = ['streams', 'tables', 'queries', 'info'];
```

Comment:
this could be derived from the keys of the tab mapping I'm suggesting in the next comment

Reply:
Used Object.keys(...) for it.